doc/guides/dts/howtos.rst

```diff
@@ -1,9 +1,12 @@
 Devicetree HOWTOs
 #################
 
-This page has instructions for getting things done with :ref:`devicetree` in
+This page has advice for getting things done with :ref:`devicetree` in
 Zephyr.
 
+.. This page could use some more love, especially giving advice to
+   driver writers about how to allocate their struct devices.
+
```

Comment:
I've been thinking it might be nice for us to have a template driver for people to start from with a bunch of this stuff flushed out.  Just an aside.